app/Console/Kernel.php

```diff
@@ -60,6 +62,14 @@ protected function schedule(Schedule $schedule): void
         $schedule->job(new SendEmptyWikiNotificationsJob)->dailyAt('21:00');
 
         $schedule->job(new UpdateQueryserviceAllowList)->weeklyOn(Schedule::MONDAY, '01:00');
+
+        $schedule->call(function () {
+            $wikis = Wiki::all();
+
+            foreach ($wikis as $wiki) {
+                dispatch(new UpdateWikiMetricDailyJob($wiki));
```

Comment:
Probably fine... I wonder if we're going to find that in production creating 1k+ jobs all at the same time is is problematic; how much worse would you find it to add a job then then spawns the other jobs over a period of time? Maybe we could even add a sleep to this closure here?